app/models/github/organization.rb

```diff
@@ -0,0 +1,127 @@
+class Github::Organization
+  include Mandate
+
+  memoize
+  def name
+    ENV["GITHUB_ORGANIZATION"] || Exercism.config.github_organization
+  end
+
+  memoize
+  def active?
+    name.present?
+  end
+
+  def remove_membership(_github_username)
+    Exercism.octokit_client.remove_organization_membership(name, username)
+  end
```

Comment:
Bug here - mismatch between `github_username` and `username`. (Also check why the test missed it)